share/vss/pedersen/vss.go

```
@@ -537,6 +554,21 @@ func (a *aggregator) VerifyDeal(d *Deal, inclusion bool) error {
 	return nil
 }
 
+// cleanVerifiers checks the aggregator's response array and creates a StatusComplaint 
+// response for all verifiers who have no response in the array.
+func (a *aggregator) cleanVerifiers() error {
```

This function does not need to return an error I guess.